NEWS.md

```diff
@@ -367,11 +367,12 @@ This section lists changes that do not have deprecation warnings.
   * `findn(x::AbstractArray)` has been deprecated in favor of `find(!iszero, x)`, which
     now returns cartesian indices for multidimensional arrays (see below, [#25532]).
 
-  * `find` now returns the same type of indices as `keys`/`pairs` for `AbstractArray`,
-    `AbstractDict`, `AbstractString`, `Tuple` and `NamedTuple` objects ([#24774]).
+  * `find` has been renamed to `findall`, and now returns the same type of indices
+    as `keys`/`pairs` for `AbstractArray`, `AbstractDict`, `AbstractString`, `Tuple`
+    and `NamedTuple` objects ([#24774], [#25545]).
     In particular, this means that it returns `CartesianIndex` objects for matrices
     and higher-dimensional arrays instead of linear indices as was previously the case.
-    Use `LinearIndices(a)[find(f, a)]` to compute linear indices.
+    Use `LinearIndices(a)[findall(f, a)]` to compute linear indice
```

Comment:
Accidentally cut off this sentence?

Reply:
Ah, thanks, that's an editor bug, it's not the first time I get this...